wgsl/index.bs

```diff
@@ -41,11 +41,32 @@ thead {
 
 ## Goals ## {#goals}
 
- * Trivially convertable to SPIR-V
- * Constructs are defined as normative references to their SPIR-V counterparts
- * All features in [SHORTNAME] are directly translatable to SPIR-V. (No polymorphism, no general pointers, no overloads, etc)
- * Features and semantics are exactly the ones of SPIR-V
- * Each item in this spec *must* provide the mapping to SPIR-V for the construct
+In no particular order:
+
+ * Text-based
+ * Readable and writable by programmers
+ * Mathematially [one-to-one](https://en.wikipedia.org/wiki/Injective_function) with SPIR-V
```

Comment:
This is backwards! We want SPIR-V to be one-to-one with WGSL, not the other way around.